libstorage/src/SystemCmd.cc

```diff
@@ -500,6 +551,34 @@ SystemCmd::checkOutput()
 }
 
 
+void
+SystemCmd::sendStdin()
+{
+    if ( ! _childStdin )
+        return;
+
+    if ( ! _stdinText.empty() )
+    {
+        string::size_type count = 0;
+        string::size_type len   = _stdinText.size();
+        int result = 1;
+
+        while ( count < len && result > 0 )
+            result = fputc( _stdinText[ count++ ], _childStdin );
+
+        _stdinText.erase( 0, count );
+        // y2deb( count << " characters written; left over: \"" << _stdinText << "\"" );
```

Comment:
Well, in principle, yes; it's a matter of habit to keep useful debugging code (albeit commented out) in case it's useful in the future again for any debugging.